<launch>
    <node name="point_cloud_clustering_node" pkg="point_cloud_clustering" type="point_cloud_clustering_node">
        <remap from="pointcloud" to="/camera_d455/depth/color/points"/>
        <param name="downsample_leaf_size" value="0.05"/>
        <param name="ransac_max_iterations" value="50"/>
        <param name="ransac_distance_threshold" value="0.1"/>
        <param name="cluster_tolerance" value="0.5"/>
        <param name="min_cluster_size" value="20"/>
        <param name="max_cluster_size" value="500"/>
        <param name="same_obj_thres" value="1"/>
        <param name="passthrough_min" value="0.0"/>
        <param name="passthrough_max" value="5.0"/>

        <param name="avg" value="10.0"/>

    </node>
</launch>